info.puredata.Pd-extended.yaml

```
@@ -0,0 +1,109 @@
+app-id: info.puredata.Pd-extended
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+default-branch: stable
+command: pd-extended
+finish-args:
+# Display
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```

```suggestion
  - --socket=fallback-x11
  - --socket=wayland
```

if it really supports wayland, we use `fallback-x11`. If it doesn't then the `wayland` socket must go.

---

I think it doesn't actually support wayland, so I removed it.